core/ROB.cpp

```diff
@@ -188,6 +194,8 @@ namespace olympia
     {
         if(getClock()->currentCycle() - last_retirement_ >= retire_timeout_interval_)
         {
+            if(reorder_buffer_.empty())
```

Comment:
@knute-sifive I had to add this else the event is scheduled forever and simulation doesnt end for some of the tests with small input like example.json file